OmsAgent/omsagent.py

```diff
@@ -931,9 +931,9 @@ def is_vm_supported_for_extension():
     The supported distros of the OMSAgent-for-Linux are allowed to utilize
     this VM extension. All other distros will get error code 51
     """
-    supported_dists = {'redhat' : ['6', '7', '8'], 'red hat' : ['6', '7', '8'], 'rhel' : ['6', '7', '8'], # Red Hat
```

Comment:
Is this discussed with Luke?

Reply:
yes, this came  a req from OMI to fix MSRC security fixes. Starting OMI release 1.7.1.* rhel 6 will no longer be supported.